tests/UI/Component/ViewControl/ViewControlTest.php

```diff
@@ -29,12 +29,12 @@ class ViewControlTest extends ILIAS_UI_TestBase
 {
     protected array $actions = [
         "ILIAS" => "http://www.ilias.de",
-        "Github" => "http://www.github.com"
+        "GitHub" => "http://www.github.com"
```

Comment:
```suggestion
        "GitHub" => "https://www.github.com"
```

Reply:
This change is not strictly related to the spelling fix, but could also be done in the same step.